package.json

```diff
@@ -23,23 +23,23 @@
   "repository": "https://github.com/sventschui/preact-ssr-prepass",
   "author": "Sven Tschui <sventschui@gmail.com>",
   "license": "MIT",
-  "husky": {
-    "hooks": {
-      "pre-commit": "prettier --write src/**/*.js typings/**/*.tsx typings/**/*.ts"
-    }
+  "lint-staged": {
+    "**/*.{js,jsx,ts,tsx}": [
+      "prettier --write src/**/*.js typings/**/*.tsx typings/**/*.ts"
+    ]
   },
   "scripts": {
-    "test": "npm run test:flow && npm run test:unit && npm run test:ts",
+    "test": "yarn test:flow && yarn test:unit",
```

Comment:
Currently the flow-tests include this really old TS-version which makes the CI fail on `test:ts`

Reply:
+1 for removing flow. I don't think anyone uses that anymore outside of FB 😬